ods_ci/tests/Tests/1000__model_serving/1009_model_serving_triton_on_keserve/1009_model_serving_triton_kserve.robot

```diff
@@ -0,0 +1,96 @@
+*** Settings ***
+Documentation     Suite of test cases for OVMS in Kserve
+Library           OperatingSystem
+Library           ../../../libs/Helpers.py
+Resource          ../../../Resources/Page/ODH/JupyterHub/HighAvailability.robot
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHModelServing.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Projects.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/DataConnections.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/ModelServer.resource
+Resource          ../../../Resources/Page/ODH/ODHDashboard/ODHDashboardSettingsRuntimes.resource
+Resource          ../../../Resources/Page/ODH/Monitoring/Monitoring.resource
+Resource          ../../../Resources/OCP.resource
+Resource          ../../../Resources/CLI/ModelServing/modelmesh.resource
+Resource          ../../../Resources/Common.robot
+Suite Setup       Triton On Kserve Suite Setup
+Suite Teardown    Triton On Kserve Suite Teardown
+Test Tags         Kserve
+
+
+*** Variables ***
+${INFERENCE_REST_INPUT_PYTORCH}=    @tests/Resources/Files/triton/kserve-triton-resnet-rest-input.json
+${PRJ_TITLE}=    ms-triton-project
+${PRJ_DESCRIPTION}=    project used for model serving triton runtime tests
+${MODEL_CREATED}=    ${FALSE}
+${PYTORCH_MODEL_NAME}=    resnet50
+${PYTORCH_RUNTIME_NAME}=    kserve-triton-pytorch-rest
+${RESOURCES_DIRPATH}=        tests/Resources/Files/triton
+${PYTORCH_RUNTIME_FILEPATH}=    ${RESOURCES_DIRPATH}/triton_pytorch_rest_servingruntime.yaml
+${EXPECTED_INFERENCE_REST_OUTPUT_FILE_PYTORCH}=       tests/Resources/Files/triton/kserve-triton-resnet-rest-output.json
+
+
+*** Test Cases ***
+
+Test PYTORCH Model Inference Via UI(Triton on Kserve)
+    [Documentation]    Test the deployment of an pytorch model in Kserve using Triton
+    [Tags]    Sanity           RHOAIENG-11561
+
+    Open Data Science Projects Home Page
+    Create Data Science Project    title=${PRJ_TITLE}    description=${PRJ_DESCRIPTION}
+    ...    existing_project=${FALSE}
+    Open Dashboard Settings    settings_page=Serving runtimes
+    Upload Serving Runtime Template    runtime_filepath=${PYTORCH_RUNTIME_FILEPATH}
```

Comment:
why don't you move the upload runtime as a part of test suite and you can remove as a part of test teardown as well

Reply:
We are uploading different runtimes for every testcase